### DISCUSSION TITLE
1.16.5

### DISCUSSION THREAD

**Discussion by NatulilakkReal** [inayakitori/4dmc#61] [General]
I just wounder if its possible to make the mod avaiable for minecraft 1.16.5 and up?
its just my computer cant go above 1.16.5 and its fine if u cant but i would be really happy if u can do that

**Comment by inayakitori** [comment]
The working version will always be on the newest Minecraft major release (e.g 1.19.x, 1.20.x). The source code difference between 1.20 and 1.16 is massive and I am not willing to dedicate my time to create and maintain it.
If anyone else is interested in making a fork and creating/maintaining a version they are more than willing to (as long as they follow the strong copyleft licensing).
Also, this is something I'd prefer if people make issues for because that's for bugs/feature requests. Discussion is for general help and discussion :3

**Reply by NatulilakkReal** [reply]
well thank you for the time to reply tho
